packages/vue-sdk/src/hooks/use-articles.ts

```diff
@@ -14,11 +16,25 @@ type Return = ReturnType<typeof useQuery<ListArticlesResponse>> & {
   articles: ArticleWithoutContent[] | undefined;
 };
 
-export const useArticles = (args?: ArticleQueryArgs): Return => {
+type ApolloQueryOptions = UseQueryOptions;
+
+export const useArticles = (
+  args?: ArticleQueryArgs,
+  searchParams?: ArticleSearchArgs,
+  apolloQueryOptions?: ApolloQueryOptions,
+): Return => {
   const contentType = buildContentType(args?.contentType);
-  const queryData = useQuery<ListArticlesResponse>(LIST_ARTICLES_QUERY, {
-    variables: { ...args, contentType },
-  });
+  const queryData = useQuery<ListArticlesResponse>(
+    LIST_ARTICLES_QUERY,
+    {
+      variables: Object.assign(
```

Comment:
Could simplified to 
```
variables: {
    ...args,
    contentType,
    ...convertSearchParamsToGQL(searchParams)
}
```
Please correct if I am wrong

Reply:
Yeah you're right that would be better. I was keeping it in line with the previously added implementations but I could update it